backend/api/v1/endpoints/auth.py

```
@@ -24,59 +22,67 @@
         raise HTTPException(status_code=401, detail="Invalid credentials")
-
+    
     token = create_access_token({"sub": user.username})
     return {"access_token": token, "token_type": "bearer"}
 
-# Protected Route
-
+# ✅ Get Current User (Check Profile Status)
 @router.get("/users/me/")
-
 async def get_current_user(token: str = Depends(oauth2_scheme), db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_current_user`

(ANN201)